src/contexts/AuthContext.tsx

```diff
@@ -107,6 +109,7 @@ export const AuthProvider: React.FC<{ appId: string }> = ({ appId, children }) =
         currentMemberId: payload?.sub || null,
         authToken,
         updateAuthToken: authToken => setAuthToken(authToken),
+        isFinishedSignUpProperty: !!payload?.isFinishedSignUpProperty,
```

Comment:
NITS: backend send boolean type already, so dosn't transform isFinishedSignUpProperty type to boolean